host-controller/src/main/java/org/jboss/as/domain/controller/operations/DomainServerLifecycleHandlers.java

```diff
@@ -314,7 +356,7 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
                     for (String serverName : processes.keySet()) {
                         final String serverModelName = serverInventory.getProcessServerName(serverName);
                         if (group == null || serversInGroup.contains(serverModelName)) {
-                            serverInventory.restartServer(serverModelName, timeout > 0 ? timeout * 1000 : timeout, model, false, suspend);
+                            serverInventory.restartServer(serverModelName, 0, model, false, suspend);
```

Comment:
Is the reason for this that the 'timeout' parameter was never part of the operation definition?

Reply:
Thanks.